a11y/Dockerfile

```
@@ -0,0 +1,43 @@
+FROM node
+
+LABEL maintainer="daniel.walford@ons.gov.uk"
+
+# Explicitly declare npm to root (docker runs as root anyway) allows global package installation for npm, needed for @axe-core/cli
+RUN npm --global config set user root
+
+# We need wget to set up the Personal Package Archive (PPA) and unzip to install the Chromedriver
+RUN apt update
+RUN apt install -y wget unzip
```

When installing packages, do it as one `RUN` command due to build caching and image layering. See the 'apt-get' section of (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run)
```suggestion
RUN apt update \
    && apt install -y wget unzip
```